### ISSUE TITLE
Markers wrong position on full screen

### ISSUE THREAD

**Issue by JavoByte** [istarkov/google-map-react#464]
Google maps include a button to toggle full screen. My markers are ok on a contained `div` but toggling full screen moves the markers as if the map didn't realize its size updated.

Contained div:
 
![captura de pantalla 2017-11-15 20 03 40](https://user-images.githubusercontent.com/4512108/32870144-2f9224da-ca40-11e7-8cfd-3122da3bb66a.png)

Full Screen:

![captura de pantalla 2017-11-15 20 04 53](https://user-images.githubusercontent.com/4512108/32870171-47eec63c-ca40-11e7-949e-1dbefa0117ec.png)

**Comment by AmarShaked** [comment]
+1

**Comment by appzter** [comment]
+1

**Comment by antoxann96** [comment]
Faced the same issue. @JavoByte do you have solution for that?

**Comment by appzter** [comment]
I figured out how to disable full screen at least if anyone is interest in that. 

Just add this function in your class 

```js
createMapOptions = maps => {
    return {
        fullscreenControl: false
    }
}
```

and then in the GoogleMapReact JSX element

```js
<GoogleMapReact
        options={this.createMapOptions}
        ...other properties
 </GoogleMapReact>
```

**Comment by itsmichaeldiego** [comment]
This was fixed in #452 and it is merged and published. Feel free to upgrade to `0.26.0` 

CC @JavoByte @AmarShaked @appzter @antoxann96
